cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,52 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+.. versionadded:: 2.7
+    Bundle dependencies was introduced in Symfony 2.7.
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inherit from them.
+
+While Composer takes care about making sure these dependencies are loaded,
+you'll also need to enable these bundles in the kernel.
+
+If your bundle is meant to be reused, bundle dependencies will complicate
+your installation documentation. This makes installation and upgrading your
+bundle more tedious for your users.
+
+You can avoid this by specifying your dependencies, this will make sure
+they are loaded, and *before* your own bundle to make sure you can extend
```

Comment:
"You can avoid this by specifying your dependencies. This will make sure that they are loaded in the kernel. It'll also make sure they are loaded *before* your own bundle, to make sure you can extend the bundle." (other than extending a bundle, it doesn't have to be loaded before afaik)